include/openthread/border_routing.h

```diff
@@ -226,6 +239,56 @@ otError otBorderRoutingSetEnabled(otInstance *aInstance, bool aEnabled);
  */
 otBorderRoutingState otBorderRoutingGetState(otInstance *aInstance);
 
+/**
+ * Configures the OMR prefix handling in the Border Routing Manager.
+ *
+ * This function offers manual administration options to explicitly configure the OMR prefix or to disable it.
+ *
+ * By default, `OT_BORDER_ROUTING_OMR_CONFIG_AUTO` is used. In this mode, the Border Routing Manager automatically
+ * selects and manages the OMR prefix. This can involve auto-generating a local prefix or utilizing a prefix obtained
+ * through DHCPv6 PD (Prefix Delegation), if the feature is enabled.
+ *
+ * The `OT_BORDER_ROUTING_OMR_CONFIG_CUSTOM` option enables the use of a user-specified OMR prefix. When this option
+ * is selected, the @p aOmrPrefix and @p aPreference parameters are used to define the custom OMR prefix and its
+ * associated preference. These parameters are ignored for other configuration modes, and @p aOmrPrefix can be `NULL`.
+ *
+ * The `OT_BORDER_ROUTING_OMR_CONFIG_DISABLE` option prevents the Border Routing Manager from adding any local or
```

Comment:
In general, an application/user can still add any prefix through the `netdata publish prefix` CLI command or related API. This prefix can be an OMR prefix. Just to check: this prefix is not managed by Border Routing Manager; so it can still be added right?  I.e. it's not prevented.

Reply:
Yes. Of course. Users can use `netdata` APIs to add directly, and it won't be impacted or prevented by the `RoutingManager`.

The `OT_BORDER_ROUTING_OMR_CONFIG_DISABLE` means the `RoutingManager` module itself won't add any OMR prefix (whether the local randomly generated one, or from DHCP6-PD).

Reply:
Agree, this is good!